tools/release_builder.py

```diff
@@ -38,53 +89,18 @@ def __init__(
         self._force_upload = os.environ.get('FORCE_ARTIFACT_UPLOAD', '').lower() == 'true'
         self._beta_release = beta_release.lower() == 'true'
 
-        name_match = re.match('.+/stub-universe-(.+).(json)$', stub_universe_url)
-        if not name_match:
-            raise Exception('Unable to extract package name from stub universe URL. ' +
-                            'Expected filename of form "stub-universe-[pkgname].json"')
+        self._stub_universe_pkg_name = self.get_package_name(stub_universe_url)
```

Comment:
@nickbp this line only seems to match `json` files? I may have missed where the `zip` files where referenced.